modules/luci-base/po/de/base.po

```diff
@@ -1237,7 +1237,7 @@ msgstr ""
 
 #: modules/luci-mod-system/luasrc/model/cbi/admin_system/leds.lua:52
 msgid "Custom flash intervall (%s)"
-msgstr ""
+msgstr "Angepasste Blink-Periode"
```

Comment:
The '(%s)` is missing and hhat sounds also weird in german is there not a nicer translation?

Reply:
Indeed, I forgot the formatting. And I do not have really better wording at the moment.